javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2633,8 +2647,24 @@ public static ExpressionTree yieldTreeGetValue(Tree yieldTree) {
      * @return true if the variableTree is declared using the {@code var} Java keyword
      */
     public static boolean isVariableTreeDeclaredUsingVar(VariableTree variableTree) {
-        JCExpression type = (JCExpression) variableTree.getType();
-        return type != null && type.pos == Position.NOPOS;
+        // The JCVariableDecl class has a field called "declaredUsingVar" that is true if the
```

Comment:
The reference to the field is confusing, as we're not accessing that field.
The referenced commit also doesn't help without further comments - that commit didn't add that field or changes it.